puttputt/logic.py

```diff
@@ -0,0 +1,223 @@
+from puttputt.models import *
+
+def profile_creation(user):
+    profile = Profile.objects.create(user=user)
+    profile.save()
+    playerInfo = PlayerInfo.objects.create(profile=profile)
+    profile.save()
+
+def get_player(user):
+    profile = None
+    player_info = None
+    try:
+        profile = Profile.objects.get(user=user)
+        player_info = PlayerInfo.objects.get(profile=profile)
+    except:
+        print("player not found!")
+
+    return {'profile': profile, 'player_info': player_info}
+
+def get_sponsor(user):
+    try:
+        profile = Profile.objects.get(user=user)
+        sponsor = SponsorInfo.objects.get(profile=profile)
+        return sponsor
+    except:
+        return None
+
+def get_manager(user):
+    try:
+        profile = Profile.objects.get(user=user)
+        manager = ManagerInfo.objects.get(profile=profile)
+        return manager
+    except:
+        return None
+
+def get_barista(user):
+    try:
+        profile = Profile.objects.get(user=user)
+        if profile.profile_type == 'BAR':
+            return profile
+        else: 
+            return None
+    except:
+        return None
+
+def tournament_for_player(player_info):
+    return player_info.tournament
+
+def tournament_for_sponsor(sponsor):
+    try:
+        tournament = Tournament.objects.get(sponsor=sponsor)
+        return tournament
+    except:
+        return None
+
+def determine_redirect_login(user):
+
+    user_type = get_user_type(user)
+
+    if user_type == 'player':
+        location = 'index'
+        # if it's a barista -> redirect them to the drinks page
```

Comment:
I was a little confused by these comments for a second. Maybe move them below the if statements rather than above?